test/controllers/unattended_controller_test.rb

```diff
@@ -373,6 +373,16 @@ class UnattendedControllerTest < ActionController::TestCase
       refute nic.build
     end
 
+    test "should accept built notifications over ipv6 and store the address" do
+      nic6 = FactoryBot.build(:nic_managed, :with_ipv6)
+      Setting[:update_ip_from_built_request] = true
```

Comment:
Does this revert the change after the test? We have a helper for `SETTINGS`:
https://github.com/theforeman/foreman/blob/9b8f759aab350ae86eef0ecf8435d6e63c39bbd9/test/test_helper.rb#L204-L214

But I'm not sure we need to a similar thing for `Setting`.

Reply:
It does not, but from looking at the code, we don't revert any of the settings. I suppose if the test relies on a specific setting, it should be explicitly set in the setup block.